game-core/src/main/java/games/strategy/engine/data/properties/ColorProperty.java

```diff
@@ -72,17 +72,13 @@ public void paintComponent(final Graphics g) {
     label.addMouseListener(new MouseListener() {
       @Override
       public void mouseClicked(final MouseEvent e) {
-        try {
-          final Color colorSelected =
-              JColorChooser.showDialog(label, "Choose color",
-                  (ColorProperty.this.color == null ? Color.black : ColorProperty.this.color));
-          if (colorSelected != null) {
-            color = colorSelected;
-            // Ask Swing to repaint this label when it's convenient
-            SwingUtilities.invokeLater(label::repaint);
-          }
-        } catch (final Exception exception) {
```

Comment:
this was flagged as an excessive catch by IDE, but nothing requires us to catch this exception and we don't really expect it from the rendering code here. The try/catch is simply removed here as a way to also get rid of the System.err